presto-docs/src/main/sphinx/security/salesforce.rst

```diff
@@ -0,0 +1,78 @@
+============================
+Salesforce Authentication
+============================
+
+Presto can be configured to enable frontend password authentication over
+HTTPS for clients, such as the CLI, or the JDBC and ODBC drivers. The
+username and password (or password and :ref:`security token` concatenation)
+are validated by having the Presto coordinator perform a login to Salesforce.
+
+This allows Presto admins to enable users to authenticate to Presto via their Salesforce
+basic credentials. This is *not* a Salesforce connector, and does not allow users to query
+Salesforce data. It's simply a means by which users can authenticate to Presto, similar
```

Comment:
It enables users to authenticate to Presto, similar...

Reply:
@mosabua , sorry I'm not quite sure I'm following this one.  I'm changing line 10, per your first comment.